builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -149,7 +156,7 @@ node {
 				flowType = "API"
 				loadServerlessConfig()
 				updateSwaggerConfig()
-			} else if (fileExists('build.lambda')) {
+			} else if (fileExists('build.lambda') || fileExists('build.function')) {
```

Comment:
Do we need a separate build.function?

Reply:
Other way around. Let's use `build.function` similar to `build.api` or `build.website` (platform agnostic).

Reply:
OR condition was given I believe to support both the older and newer functions which has function and lambda

Reply:
@devsatishm Will that not affect existing functions?

Reply:
What are the older services in question? Jazz doesn't support seamless upgrades yet. 
If we plan to use build.function everywhere, update aws function templates (rename file build.lambda -> build.function).

Reply:
Right. Especially in the internal context it might be an issue. Probably not as much in OSS. But always good to make it as seamless as possible at least going forward.

Reply:
Let's address this as part of next release. This is not a functionality change, rather a code consistency issue.